README.md

```diff
@@ -1,7 +1,126 @@
-# Converting NeuroML2/LEMS to & from SBML
-
+# SBML Showcase
 [![Continuous build using OMV](https://github.com/OpenSourceBrain/SBMLShowcase/actions/workflows/omv-ci.yml/badge.svg)](https://github.com/OpenSourceBrain/SBMLShowcase/actions/workflows/omv-ci.yml) [![Testing non OMV scripts](https://github.com/OpenSourceBrain/SBMLShowcase/actions/workflows/non-omv.yml/badge.svg)](https://github.com/OpenSourceBrain/SBMLShowcase/actions/workflows/non-omv.yml)
 
+This repository contains files and instructions for testing validity of Systems Biology Markup Language (SBML) and Simulation Experiment Description Markup Language (SED-ML) files, and their compatibility with different simulations engines. 
+
+## Results tables
+
+### SBML
+- [LEMS_NML2_Ex9_FN](SBML\tests\results_compatibility_biosimulators.md)
```

Comment:
@stellaprins Many of these links have backslashes.. which is fine on Windows, but use forward slashes so the links work on Github: https://github.com/OpenSourceBrain/SBMLShowcase/blob/feat-update-readme/README.md